agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -436,7 +454,7 @@ else
     unsupported
 fi
 
-download_yamls
+#download_yamls
```

Comment:
any particular reason this was commented out?

Reply:
thanks .. good finding - this was commented out only for some testing and forgot to remove the comment ...